omod/src/test/java/org/openmrs/module/fhir2/providers/FhirResourceProviderTest.java

```diff
@@ -0,0 +1,10 @@
+package org.openmrs.module.fhir2.providers;
+
+import org.hl7.fhir.instance.model.api.IBaseBundle;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+import java.io.UnsupportedEncodingException;
+
+public interface FhirResourceProviderTest {
+    IBaseBundle readBundleResponse(MockHttpServletResponse response) throws UnsupportedEncodingException;
```

Comment:
@ibacher I have added an interface like this.

Reply:
@VaishSiddharth Sorry... I was apparently leading you astray. [This commit](https://github.com/openmrs/openmrs-module-fhir2/pull/158/commits/88232ddf1a43567bd4b11e0e1cbe88894bbafeee) has the refactoring necessary for this. I left some errors in the DSTU3 test classes though. See if you can work them out.